source/_patterns/04-templates/basic-page/_basic-page.scss

```diff
@@ -10,18 +10,18 @@
     position: relative;
   }
   @include media-breakpoint-up(xl) {
-    width: 1200px;
+    width: rem-calc(1200px);
   }
 
   &__inner {
-    padding: 50px 20px;
+    padding: rem-calc(50px 20px);
 
     @include media-breakpoint-up(md) {
-      padding: 50px;
+      padding: rem-calc(50px);
     }
 
     p {
-      line-height: 2;
+      line-height: rem-calc(32px);
```

Comment:
The `line-height` value without units is a multiplier of the font-size, so no need to convert to rem.

Reply:
@illepic Goddangit this is some knowledge that dropped to the recesses of my brain. Time to go rewrite all of my stuff.